explorer-api/src/Explorer/Web/Server.hs

```diff
@@ -65,13 +67,16 @@ import qualified Data.Text.Encoding as Text
 import qualified Network.Wai.Handler.Warp as Warp
 import qualified Servant
 
-runServer :: IO ()
-runServer = do
-  putStrLn "Running full server on http://localhost:8100/"
-  pgconfig <- readPGPassFileEnv
+runServer :: WebserverConfig -> PGConfig -> IO ()
+runServer webserverConfig pgConfig = do
+  let warpSettings = toWarpSettings webserverConfig
+      pgurl = pgcHost pgConfig <> ":" <> pgcPort pgConfig
+      warpUrl = show (Warp.getHost warpSettings) <> ":" <> show (Warp.getPort warpSettings)
+  putStrLn $ "Connecting to database at " <> unpack pgurl
+  putStrLn $ "Running full server on " <> warpUrl
```

Comment:
why are these not inside the `runStdoutLoggingT`?

Reply:
Yeah, good suggestion. I have separate branch I'm putting some housekeeping stuff on as I go. I'll do that there and make a separate PR.

Reply:
Agree with @shmish111